aim/web/ui_v2/src/services/models/metrics/metricsAppModel.ts

```diff
@@ -36,6 +36,10 @@ import { CurveEnum, ScaleEnum } from 'utils/d3';
 import getObjectPaths from 'utils/getObjectPaths';
 import getTableColumns from 'pages/Metrics/components/TableColumns/TableColumns';
 import DASH_ARRAYS from 'config/dash-arrays/dashArrays';
+import { IBookmarkFormState } from '../../../types/pages/metrics/components/BookmarkForm/BookmarkForm';
```

Comment:
You can also import with absolute path like 'types/pages/metrics/components/BookmarkForm/BookmarkForm' to avoid long path name